pkg/crc/constants/constants_windows.go

```diff
@@ -11,4 +16,32 @@ const (
 	TrayShortcutName    = "tray-windows.lnk"
 )
 
-var TrayBinaryPath = filepath.Join(CrcBinDir, "tray-windows", TrayBinaryName)
+var (
+	TrayBinaryDir  = filepath.Join(CrcBinDir, "tray-windows")
+	TrayBinaryPath = filepath.Join(TrayBinaryDir, TrayBinaryName)
+)
+
+func GetHomeDir() string {
+	// if not running in interactive mode, it is the daemon service
+	// in that case we have the user's home directory as the third
+	// argument in the service executable invocation
+	isIntSess, err := svc.IsAnInteractiveSession()
+	if err == nil {
+		if !isIntSess && len(os.Args) > 2 {
+			return os.Args[2]
+		}
+	}
+
+	if homeDrive, homePath := os.Getenv("HOMEDRIVE"), os.Getenv("HOMEPATH"); len(homeDrive) > 0 && len(homePath) > 0 {
+		homeDir := filepath.Join(homeDrive, homePath)
+		if _, err := os.Stat(homeDir); err == nil {
+			return homeDir
+		}
+	}
+	if userProfile := os.Getenv("USERPROFILE"); len(userProfile) > 0 {
+		if _, err := os.Stat(userProfile); err == nil {
+			return userProfile
+		}
+	}
```

Comment:
@anjannath doesn't this also replaced with `os.UserHomeDir()` as per doc it does return USERPROFILE in case of windows.

Reply:
@praveenkumar Ahh, sorry i missed it thought it was getting the homedir with some other logic, changed to use `os.UserHomeDir()`